giskard_vision/landmark_detection/tests/base.py

```diff
@@ -256,6 +265,7 @@ class TestDiff:
     metric: Metric
     threshold: float
     relative: bool = True
+    metric_for_examples: Optional[Metric] = None
```

Comment:
as we discussed, let's create a new method in the `Metric` class that returns a sorted list of indices (from worse to best)